### PR TITLE
Added isFullWidth prop to SFChipListField

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: src/Components/SFChipsListField/SFChipsListField.tsx [hunk 1]
```diff
@@ -95,6 +95,7 @@ export type ChipFieldValueType = {
 export interface SFChipsListFieldProps {
   chipSize?: 'small' | 'medium';
   chipDisplay?: 'inline' | 'block';
+  isFullWidth?: boolean;
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.tsx]: changed to isInputFullWidth

**Review by pdcastano — CHANGES_REQUESTED** [review]
The story is not working for the inputMinWidth prop. Try to find a way to solve the issue.

File: src/Components/SFChipsListField/SFChipsListField.tsx [hunk 2]
```diff
@@ -85,6 +92,15 @@ const chipsDisplay = makeStyles({
...
 });
 
+function StyledTextField(
+  props: TextFieldStylesProps &
+    Omit<SFTextFieldProps, keyof TextFieldStylesProps>
+): React.ReactElement {
+  const { minWidth, ...other } = props;
+  const classes = useTextFieldStyles(props);
+  return <SFTextField className={classes.root} {...other} />;
+}
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.tsx]: done

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: src/Components/SFChipsListField/SFChipsListField.tsx [hunk 3]
```diff
@@ -51,22 +51,29 @@ const StyledAutoComplete = withStyles((theme: Theme) => ({
   }
 }))(Autocomplete);
 
-const StyledTextField = withStyles({
+type minWidthStyle = number | 'auto' | 'full-width';
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.tsx]: done

**Pull request merged**